1.  FileManger file created for documents directory

v605690 opened the pull request:
1.  FileManager file created for document directory
2.  StampStore file created with incorporated new struct and functions to save and load data; plist used
3.  Create two unit test to validate test

Had a difficult time on assignment 7, confused at times. The videos helped but I felt as though was not clear enough
 
 I ended up creating a new file StampStore and incorporated a new struct along with saving and load data but not the preferred way to do things but wanted show some progress.

v605690 committed:
1.  FileManger file created for documents directory
2.  StampStore file created with incorporated new struct and functions to save and load data; plist used
3.  Create two unit test to validate test

orgcottontee commented on PatelDLTGroupStamps/Models/StampStore.swift:
```diff
@@ -0,0 +1,76 @@
+let stampsPListURL = URL(fileURLWithPath: "Stamp", relativeTo: FileManager.documentsDirectoryURL.appendingPathExtension("plist"))
+
+// MARK: Assignment 1: Saving data to Plist only
+private func savePListStamps() {
```
You never called this anywhere, so the document directory is empty (it didn't save).

orgcottontee commented on PatelDLTGroupStamps/Models/StampStore.swift:
```diff
@@ -0,0 +1,76 @@
+  }
+}
+// MARK: Assignment 2: Loading the saved data
+private func loadPListStamps() {
```
Therefore, there are no files you can load.

v605690 replied:
Thank You, can you assist so that I can re-submit?

Let me see what I can do on second attempt.

-pravin

v605690 replied:
Hi, I have updated the project app.

 Created a new StampStore and StampPListView swift file also incorporated a new struct along with saving and loading data but not the preferred way to do things.

  Not sure if the the output is correct or not by running the app.

  Please note I have updated the Hiplatelics.swift to start the view on StampPListView.

orgcottontee reviewed (approved):
Hi Pravin! 

- The save and load functions are correct, but they were never called. 
- Your unit did not test saving and loading of your data.

You get needs improvement. You may resubmit for a passing grade!

v605690 committed:
Created a new StampStore and StampPListView swift file also incorporated a new struct along with saving and loading data but not the preferred way to do things.

  Not sure if the the output is correct or not by running the app.

  Please note I have updated the Hiplatelics.swift to start the view on StampPListView.

v605690 committed:
Add UI Tests
